pkg/proc/gdbserial/gdbserver.go

```diff
@@ -895,7 +896,11 @@ continueLoop:
 		// NOTE: because debugserver will sometimes send two stop packets after a
 		// continue it is important that this is the very first thing we do after
 		// resume(). See comment in threadStopInfo for an explanation.
-		p.updateThreadList(&tu)
+		var jstopinfo map[int]stopPacket
+		if p.conn.isDebugserver { // TODO(deparker): Can we support this on RR backend?
```

Comment:
RR doesn't write jstopinfo, it's a debugserver exclusive AFAIK. This can be removed.